File: Tema 2 - Recogida de datos por teclado mediante formularios/Ejercicios/ejercicio4 sume reste multiplique y divida/archives/4.py
import sys

# Obtener los argumentos de la línea de comandos
num1 = int(sys.argv[1])
num2 = int(sys.argv[2])
suma = num1+num2
resta = num1-num2
multiplicacion = num1*num2
division = "Indefinido" if num2 == 0 else round(num1/num2, 2,)

# Imprimir el resultado
print(f"""suma:<br>{num1} + {num2} = {suma}
			<br><br>resta:<br>{num1} - {num2} = {resta}
			<br><br>multiplicacion:<br>{num1} * {num2} = {multiplicacion}
			<br><br>division:<br>{num1} / {num2} = {division} """)

File: Tema 2 - Recogida de datos por teclado mediante formularios/Ejercicios/ejercicio4 sume reste multiplique y divida/archives/4.php
<?php
$num1 = $_POST["num1"];
$num2 = $_POST["num2"];
$suma = $num1 + $num2;
$resta = $num1 - $num2;
$multiplicacion = $num1 * $num2;
$division = ($num2 != 0) ? round($num1 / $num2, 2) : "Infinity";

$mensaje = "suma:<br>$num1 + $num2 = $suma
			<br><br>resta:<br>$num1 - $num2 = $resta
			<br><br>multiplicacion:<br>$num1 * $num2 = $multiplicacion
			<br><br>division:<br>$num1 / $num2 = $division";

echo $mensaje;















/* if ($metodo == "php") { */
	// Sumar utilizando PHP
/* 	$resultado = $num1 * $num2; */
/* 	echo "la multiplicacion de <br> $num1 * $num2 es: ".$resultado;
} elseif ($metodo == "python") { */
	// Sumar utilizando Python
/* 	$output = shell_exec("python 1.py " . $num1 . " " . $num2);
	echo $output;
} else {
	echo "Método no válido";
} */
